.github/workflows/deps.yml

```diff
@@ -0,0 +1,45 @@
+# This workflow will build a Java project with Maven, and cache/restore any dependencies to improve the workflow execution time
+# For more information see: https://help.github.com/actions/language-and-framework-guides/building-and-testing-java-with-maven
+
+name: Deps CI with Maven
+
+concurrency:
+  group: ${{ github.workflow }}-${{ github.ref }}
+  cancel-in-progress: true
+
+on:
+  push:
+    branches:
+      - "master"
+      - "0.**"
```

Comment:
nit: here add `0.**` branch, isn't `master` enough for this, for the pattern`0.**`, we need to update this if the version become `1.xx`